Makefile

```diff
@@ -75,6 +75,10 @@ hypershift-operator:
 control-plane-operator:
 	$(GO_BUILD_RECIPE) -o $(OUT_DIR)/control-plane-operator ./control-plane-operator
 
+.PHONY: control-plane-pki-operator
```

Comment:
With https://github.com/openshift/hypershift/pull/3193/commits/182804c9d0bffe49805e14b32df5563ac419aeee#r1395979845 then there's no need to adhoc build here since this will be a cpo subcommand